ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala

```diff
@@ -316,37 +316,39 @@ object HttpServiceTestFixture extends LazyLogging with Assertions with Inside {
   def jwtForParties(
       actAs: List[String],
       readAs: List[String],
-      ledgerId: String,
+      ledgerId: Option[String] = None,
       withoutNamespace: Boolean = false,
```

Comment:
I know this is existing code, but in general I don't think it's great to have these 'boolean switch arguments' in a function, and on top of that all defaulted. (It's so easy to forget to set a boolean flag somewhere, and what it means, in a specific case, or to understand what is the default, where it is used) 

Would rather prefer a few versions of something like this that just creates the jwt that you need for admin, or without a namespace, etc. Or maybe even a builder.

Reply:
I don't like boolean flags for this kind of stuff as well as they tend to make code brittle over time, as you end up adding more flags and have a combinatorial explosions of conditionals in your code. In this context it's probably acceptable though.

Reply:
I agree booleans are bad, and this PR arguably makes it worse here by adding one more branch, but since the problem is mostly with existing code I'd suggest making that refactoring as a separate PR.

Reply:
It was more something I noticed in a few places, it's not a blocker for the PR.